flang/lib/Evaluate/intrinsics.cpp

```diff
@@ -1403,6 +1403,11 @@ static const IntrinsicInterface intrinsicSubroutine[]{
             {"stat", AnyInt, Rank::scalar, Optionality::optional,
                 common::Intent::Out}},
         {}, Rank::elemental, IntrinsicClass::atomicSubroutine},
+    {"chdir",
+        {{"dir", AnyChar, Rank::anyOrAssumedRank},
```

Comment:
Please use the  "NAME" and "STATUS" GCC documentation names as these are the names that may be printed in user error message, so they should relate to the specifications.